sipa/utils/__init__.py

```diff
@@ -159,6 +159,25 @@ def meetingcal():
     return next_meetings
 
 
+def subscribeStatusPage(url: str, email: str) -> bool | None:
```

Comment:
Please use snail-case function names according to [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names):
```suggestion
def subscribe_to_status_page(url: str, email: str) -> bool | None:
```